spec/models/order_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  describe "validations" do
```

Comment:
No order validations? Did you get your code coverage up without explicitly testing the order model validations?